packages/phone-number-privacy/combiner/src/database/wrappers/number-pairs.ts

```diff
@@ -1,7 +1,7 @@
-import { ErrorMessage } from '../../common/error-utils'
+import { ErrorMessage } from '@celo/phone-number-privacy-common'
 import logger from '../../common/logger'
 import { getDatabase } from '../database'
-import { NUMBER_PAIRS_COLUMN, NUMBER_PAIRS_TABLE, NumberPair } from '../models/numberPair'
+import { NumberPair, NUMBER_PAIRS_COLUMN, NUMBER_PAIRS_TABLE } from '../models/numberPair'
```

Comment:
I suspect this will fail the linter :)
VS Code's ordering differs from prettier. I don't know of a good way to fix this so I usually reorder them manually then to `command+shift+p` then choose 'Save without formatting'

Reply:
Ah I was wondering why it kept complaining with the changes it was auto-making.. thanks!

Reply:
I haven't used VSCode in a while but I'm certain there's a `prittier` plugin that can be used for autoformatting so that it abides by the same rules as the linter.